sdx-submitter.yml

```diff
@@ -0,0 +1,6 @@
+port: 5672
+host: "localhost"
+exchange: "message"
+vhost: "%2f"
```

Comment:
One question about this.  Is what this %2f means obvious?  It's probably fine, and It's just me being a newbie at rabbit stuff ^^ but I thought I might as well ask the question

Reply:
It does look odd I know . Rabbit has a virtual host that is specified in the connection string . The default being "/" . %2f is the URL encoded value for that . It looks arbitrary on first viewing though